superset-frontend/packages/superset-ui-core/src/query/getMetricLabel.ts

```diff
@@ -19,17 +19,23 @@
 
 import { QueryFormMetric, isSavedMetric, isAdhocMetricSimple } from './types';
 
-export default function getMetricLabel(metric: QueryFormMetric): string {
+export default function getMetricLabel(
+  metric: QueryFormMetric,
+  index?: number,
+  queryFormMetrics?: QueryFormMetric[],
+  verboseMap: Record<string, string> = {},
```

Comment:
Nit. Does it need to be defaulting to an empty object or can make this also optional and check for both undefined and empty?